backend/src/zimfarm_backend/utils/token.py

```diff
@@ -99,7 +103,17 @@ def get_public_key_fingerprint(public_key: RSAPublicKey) -> str:
 
 
 def load_rsa_public_key(key: str) -> RSAPublicKey:
```

Comment:
Looks like we do not check that key passed is RSA, while we do expect this. Should we add an explicit check?

Reply:
So, throughout the development, I had been thinking that the keys were passed in the PEM format, not knowing that it was the openSSH format. It was during the design of the frontend that I realized the keys were `.pub` files which were often in openSSH format. By adding it as the first format to try, the system can allow users to upload both formats.